kernel/fs/fs.go

```diff
@@ -70,8 +70,8 @@ func (s *Service) Initialize() {
 		}
 	}
 
-	s.fsys.MkdirAll("cmd/sauce", 0755)
-	fsutil.WriteFile(s.fsys, "cmd/sauce/main.go", []byte("package main\nimport \"fmt\"\nfunc main() {\n\tfmt.Println(\"Hello World!\")\n}"), 0644)
+	fs.MkdirAll(s.fsys.FS, "cmd/sauce", 0755)
+	fsutil.WriteFile(s.fsys.FS, "cmd/sauce/main.go", []byte("package main\nimport \"fmt\"\nfunc main() {\n\tfmt.Println(\"Hello World!\")\n}"), 0644)
```

Comment:
These two lines can just be deleted. They were for a test and accidentally committed.